ZCRMiOS/Classes/ZCRMTag.swift

```diff
@@ -12,13 +12,27 @@ open class ZCRMTag : ZCRMTagDelegate
     var modifiedBy : ZCRMUserDelegate = USER_MOCK
     var modifiedTime : String = APIConstants.STRING_MOCK
     
-    init( tagName : String )
+    internal init( tagName : String )
     {
         super.init( tagId : APIConstants.INT64_MOCK, tagName : tagName, moduleAPIName : APIConstants.STRING_MOCK )
     }
     
-    init()
+    internal init()
     {
         super.init( tagId : APIConstants.INT64_MOCK, moduleAPIName : APIConstants.STRING_MOCK)
     }
+    
+    public func update( updateTag : ZCRMTag, completion : @escaping ( Result.DataResponse< ZCRMTag, APIResponse > ) -> () )
```

Comment:
Whats this update method? if its update of tag, why does it require an argument?